osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SliderBody.cs

```diff
@@ -51,6 +53,24 @@ public Color4 AccentColour
             }
         }
 
+        private Color4 borderColour = Color4.White;
+        /// <summary>
+        /// Used to colour the path border.
+        /// </summary>
+        public new Color4 BorderColour
+        {
+            get { return borderColour; }
+            set
+            {
+                if (borderColour == value)
+                    return;
+                borderColour = value;
+
+                if (LoadState == LoadState.Ready)
+                    Schedule(reloadTexture);

```

Comment:
don't think the schedule here is required. we shouldn't be doing cross-thread border colour changes post-load.

Reply:
This is the same thing as what `AccentColour` does. This is probably done so we don't reload the texture multiple times prior to load() being run (which does `reloadTexture` itself).